test-requirements.txt

```diff
@@ -1 +1,4 @@
 tox==2.7.0
+pytest==3.1.2
```

Comment:
No, not really. Just in the spirit of exactly pinning dependencies. Later on, one could outsource dependency updating to a bot like https://pyup.io/ which will create PRs when a dependency could be updated. However, can change to `3.1.x` or even  `3.x.x` (not sure about the exact syntax though)